packages/core/src/store.ts

```diff
@@ -48,6 +48,6 @@ export default (initialState: any, middleware: [] = []): Store<RootState> => {
   return createStore(
     rootReducer,
     initialState,
-    v(applyMiddleware(...middleware))
+    v(applyMiddleware(...middleware, thunk))
```

Comment:
we now need the thunk middleware. I think we need to mention that, if you pass your own store to it